source/includes/steps-create-system-dsn.yaml

```diff
@@ -1,75 +1,73 @@
-title: Start the Microsoft ODBC Data Sources program.
+title: Start the Microsoft ODBC Data Sources application.
 ref: start-odbc
 level: 4
 stepnum: 1
 content: |
-  Choose the program version (64-bit or 32-bit) which is
-  appropriate for your system and ODBC driver version.
+  Choose the edition (64-bit or 32-bit) compatible with your platform
+  and ODBC driver edition.
 ---
-title: Select the ``System DSN`` tab. 
+title: Add a System DSN.
 ref: select-tab
 level: 4
 stepnum: 2
----
-title: Click the ``Add`` button. 
-ref: click-add
-level: 4
-stepnum: 3
 content: |
-   .. figure:: /images/bi-connector/windows-odbc-administrator.png
+  .. figure:: /images/bi-connector/windows-odbc-administrator.png
       :alt: Screenshot of the Windows ODBC Administrator application
       :figwidth: 550px
+
+  a. Click :guilabel:`System DSN`.
+  b. Click :guilabel:`Add...`
 ---
-title: Select ``MongoDB ODBC 1.0 Unicode Driver`` from the list of available drivers.
+title: From the list of available drivers, click :guilabel:`MongoDB ODBC 1.0 Unicode Driver`.
 ref: select-odbc
 level: 4
-stepnum: 4
+stepnum: 3
 ---
-title: Fill in the necessary form fields. 
+title: Provide the connection settings.
```

Comment:
Prefer "form fields" but like the use of "connection" in your update. Fill in the connection form fields

Reply:
Fixed.